snakemake_executor_plugin_googlebatch/executor.py

```diff
@@ -202,8 +223,16 @@ def run_job(self, job: JobExecutorInterface):
         group.permissive_ssh = True
 
         # This includes instances (machine type) boot disk and policy
+        # Also preemtion
         policy = self.get_allocation_policy(job)
 
+        # If we have preemption for the job and retry, update task retries with it
+        retries = self.workflow.remote_execution_settings.preemptible_retries
```

Comment:
The task has retries, and I figure that parameter corresponds to "preemtible retries" when that is set. Note that without it, it's just the number of retries regardless of preemptible.